test/rules/grouped-imports/bad-order.ts.lint

```diff
@@ -0,0 +1,6 @@
+import {bar} from '../bar';
+
+import {foo} from 'foo';
+~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources of different groups must be sorted by: libraries, parent directories, current directory]
```

Comment:
please add a test that has both kinds of error: unnecessary blank line and wrong order